javascript/src/test/components/Courses/CourseList.test.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { render } from "@testing-library/react";
+// import CourseList from "main/components/Courses/CourseList";
```

Comment:
Can we remove this comment?

Reply:
I will remove this line in the next commit